packages/web3-react/src/helpers/regex.ts

```diff
@@ -0,0 +1,5 @@
+const pattern = new RegExp(
+  /(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]+\.[^\s]{2,}|www\.[a-zA-Z0-9]+\.[^\s]{2,})/,
+);
+
+export const isUrl = (url: string): boolean => pattern.test(url);
```

Comment:
It is not obvious why we have `isUrl` checker in the file named `regex`. I assume that this file is intended for adding some another regex-check-functions in the future, but maybe it is better to separate such checkers to separate files.
So, this file can be named as `isUrl.ts` and the export can be default.

Reply:
Totally agree.